index.html

```diff
@@ -5,13 +5,11 @@
 <h2>Upcoming Events?</h2>
 
 <ol id="events">
-  <li><a href="#au_feb_2022">Adelaide, Australia, February 4th–7th, 2022</a></li>
-  <li><a href="#us_september_2020">North Cascades National Park, Washington, USA, September 2020</a></li>
+  <li><a href="#au_oct_2024">Warrnambool, Australia, October 18th-21st, 2024</a></li>
 </ol>
 
-<p class="placeholder">Rails Camps are becoming regular fixtures in countries like Australia, New Zealand, UK, USA, Denmark and Poland. It would be awesome to see more happen elsewhere. Rails Camps happen because people make them happen. People like you.</p>
+<p class="placeholder">Ruby Retreats are becoming regular fixtures in countries like Australia, New Zealand, UK, USA, Denmark and Poland. It would be awesome to see more happen elsewhere. Ruby Retreats happen because people make them happen. People like you.</p>
```

Comment:
Should this maybe say 
```suggestion
<p class="placeholder">Rails Camps and Ruby Retreats are becoming regular fixtures in countries like Australia, New Zealand, UK, USA, Denmark and Poland. It would be awesome to see more happen elsewhere. Rails Camps and Ruby Retreats happen because people make them happen. People like you.</p>
```